beta/src/sidebarHome.json

```diff
@@ -15,7 +15,7 @@
           "title": "Community",
           "path": "/community",
           "routes": [{
-            "title": "Acknowledgements",
+            "title": "Acknowledgments",
```

Comment:
If you change this, please also change the link, the title on the page, etc.

Reply:
Right, changed it also

Reply:
Right!
Changed everything under beta.